application/org.openjdk.jmc.flightrecorder.ui/src/main/resources/org/openjdk/jmc/flightrecorder/ui/messages/internal/messages.properties

```diff
@@ -509,7 +510,7 @@ STACKTRACE_VIEW_LAST_FRAME=Last Frame
 STACKTRACE_VIEW_LAYOUT_OPTIONS=Layout Options
 STACKTRACE_VIEW_SELECTION=Stack Trace View Selection
 STACKTRACE_VIEW_TREE_VIEW=Tree View
-STACKTRACE_VIEW_PERCENTAGE_BY_DURATION=Show Percentage By Duration
+STACKTRACE_VIEW_PERCENTAGE_BY_DURATION=Show Duration
```

Comment:
Without the 'Percentage By' portion in the string, should this be something like 'STACKTRACE_VIEW_DURATION' ?

Reply:
Done.